### PR TITLE
feat : label

### PR DESCRIPTION
**Pull request by arpit01923**
Functionality:
- you should add labels while making notes
- on labels page you should filter out note according to label

Files to check:
- src/pages/home/home.jsx
- src/pages/labels/labels.jsx

### CODE REVIEW

File: src/components/navbar/navbar.jsx [hunk 1]
```diff
@@ -10,11 +10,11 @@ export const Navbar = () =>{
           <h1 className="text">Note App</h1>
         </span>
 
-        <input
+        {/* <input
```
Comment (deepak29-git) [inline src/components/navbar/navbar.jsx:13]: remove commented code

File: src/pages/trash/trash.jsx [hunk 2]
```diff
@@ -10,12 +10,13 @@ export const Trash = () => {
         <Sidebar />
         <main className="content">
           {noteArrayState.trash.map(
-            ({ _id, title, text, color, date, time }, index) => (
+            ({ _id, title, text, tags, color, date, time }, index) => (
               <NoteContainer
                 key={index}
```
Comment (deepak29-git) [inline src/pages/trash/trash.jsx]: use id instead of index

**Review by deepak29-git — APPROVED** [review]
folder structure is good

**Review by manojncsoni007 — APPROVED** [review]
Everything looks good to me
- Clean code and simple project structure

**Review by aniketsingh13 — APPROVED** [review]
code structure look good
code is easy to read

**Pull request merged**